src/main/java/com/bookstore/dto/order/CreateOrderRequestDto.java

```diff
@@ -0,0 +1,18 @@
+package com.bookstore.dto.order;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import java.util.HashSet;
+import java.util.Set;
+import lombok.Data;
+
+@Data
+public class CreateOrderRequestDto {
```

Comment:
CreateOrderRequestDto should include only String shippingAddress according to the task description